test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -23,8 +28,14 @@
     countParam: $measurePods
     collectMetrics: true
   workloads:
+  - name: 5Nodes
+    labels: [integration-test, fast, short]
+    params:
+      initNodes: 5
+      initPods: 5
+      measurePods: 10
   - name: 500Nodes
-    labels: [integration-test, fast]
+    labels: [fast]
```

Comment:
why not keep `integration-test` so it can run on ci-kubernetes-integration-master? Similar question for the rest of the "mid size" tests.

Reply:
I'm not sure if it would give us any noticeable advantage. We still have periodic performance tests with larger workloads that could catch the performance issues. As @pohly said above, we only treat integration tests as a check if the test cases are properly configured and if scheduler doesn't have any (obvious) errors when running with those configs.

Reply:
Sounds good.

Reply:
There's one small future benefit of keeping mid-sized workloads enabled for ci-kubernetes-integration-master: at some point I want to enable race detection for integration tests. This cannot be done right now because there are known races in the apiserver and I don't want to deal with a constant stream of known failures.

Once those are fixed and race detection is enabled, mid-sized workloads might be more likely to trigger races than the small ones.

Reply:
Okay, so do we want to enable mid-sized workloads for ci-kubernetes-integration-master now?

Reply:
Tentatively yes? I'm on the edge.

Reply:
If we have the compute, I say let's run them.

Reply:
Re-added `integration-test` label for the mid size workloads.